### PR TITLE
docs: examples

### PR DESCRIPTION
**Pull request by diasdavid**
> WIP

### CODE REVIEW

**Comment by VictorBjelkholm** [comment]
PR says WIP, would be useful to have a note that the examples are WIP as well :)

**Comment by diasdavid** [comment]
@VictorBjelkholm I started this work today. I like to keep open PRs as I develop stuff. Examples don't exist yet as they require me to change other things to make them work as I want to (i.e improve API, docs, fix errors, etc). Sorry for the confusion!

**Comment by diasdavid** [comment]
@VictorBjelkholm it is coming https://github.com/libp2p/js-libp2p/pull/99/commits/19d67b920e0505e847ef117199f51f677de79263 ;)

File: examples/README.md [hunk 1]
```diff
@@ -4,8 +4,23 @@ In this folder, you can find a variety of examples to help you get started in us
 
 Let us know if you find any issue or if you want to contribute and add a new tutorial, feel welcome to submit a PR, thank you!
 
-## Examples
+## Understanding how libp2p works
 
+- [Transports](./transports)
```
Comment (diasdavid) [inline examples/README.md]: `Transports` example is now ready for review 🌟

File: examples/README.md [hunk 2]
```diff
@@ -4,8 +4,23 @@ In this folder, you can find a variety of examples to help you get started in us
...
+- [Stream and Protocol Muxing](./protocol-and-stream-muxing)
```
Comment (diasdavid) [inline examples/README.md]: `Protocol and Stream Muxing` is also ready for review 🌟

File: examples/README.md [hunk 3]
```diff
@@ -4,8 +4,23 @@ In this folder, you can find a variety of examples to help you get started in us
 
 Let us know if you find any issue or if you want to contribute and add a new tutorial, feel welcome to submit a PR, thank you!
 
-## Examples
+## Understanding how libp2p works
 
+- [Transports](./transports)
+- [Protocol and Stream Muxing](./protocol-and-stream-muxing)
```
Comment (diasdavid) [inline examples/README.md]: `Protocol and Stream Muxing` is also ready for review 🌟

File: examples/README.md [hunk 4]
```diff
@@ -4,8 +4,23 @@ In this folder, you can find a variety of examples to help you get started in us
...
+- [Encrypted Communications](./encrypted-communications)
```
Comment (diasdavid) [inline examples/README.md]: Another one _bytes_ the dust! 🌟

**Comment by diasdavid** [comment]
Let's do an experiment and open these up through reddit for the community to jump on and review :)

**Pull request merged**